resources_for_CI/Reporter.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+
+from github import Github
```

Comment:
I would add a comment either here or in a readme somewhere explaining exactly what this is doing regarding reading of comments/parsing comments and how it decides to update them.